src/browser/app.rs

```diff
@@ -28,24 +28,20 @@ impl App {
 
     let stderr = Stderr::get();
 
-    let context = canvas
```

Comment:
We want to still be able to use the CPU renderer in the app until the GPU has reached feature parity, so we should default to using the CPU, and only use the GPU if the URL ends with `#gpu`.